scripts/run-every-smoke-test.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
```

Comment:
Convenience script for developers to run every smoke test generated under `SmokeTests/` with overall test summary printed at the end.